pkg/runner/case_conversion.go

```diff
@@ -0,0 +1,147 @@
+// Based on https://github.com/iancoleman/strcase
+package runner
```

Comment:
Should this move to a util package?

Reply:
What about a conversion package or a `paramconv` package(like `strconv` from standard library)?

Reply:
done